cocotb/config.py

```diff
@@ -50,6 +51,49 @@
 makefiles_dir = os.path.join(os.path.dirname(cocotb.__file__), "share", "makefiles")
 
 
+def help_vars_text():
+    if "dev" in cocotb.__version__:
+        doclink = "https://docs.cocotb.org/en/latest/building.html"
+    else:
+        doclink = "https://docs.cocotb.org/en/v{}/building.html".format(cocotb.__version__)
+
+    # NOTE: make sure to keep "helpmsg" aligned with documentation/source/building.rst
+    # Also keep it at 80 chars.
+    helpmsg = textwrap.dedent("""
```

Comment:
```suggestion
    helpmsg = textwrap.dedent("""\
```
to remove the leading newline

Reply:
Done, thx.